lib/ohai/plugins/packages.rb

```diff
@@ -22,63 +22,78 @@
   provides "packages"
   depends "platform_family"
 
+  WINDOWS_ATTRIBUTE_ALIASES = {
+    "DisplayVersion" => "version",
+    "Publisher" => "publisher",
+    "InstallDate" => "installdate"
+  }
+
   collect_data(:linux) do
-    if configuration(:enabled)
```

Comment:
It looks like this plugin was disabled by default [here](https://github.com/chef/ohai/pull/717). Are you intentionally reverting that?